_IBC/composable-osmosis.json

```diff
@@ -0,0 +1,32 @@
+{
+    "$schema": "../ibc_data.schema.json",
+    "chain_1": {
+      "chain_name": "composable",
+      "client_id": "07-tendermint-6",
+      "connection_id": "connection-6"
+    },
+    "chain_2": {
+      "chain_name": "osmosis",
+      "client_id": "07-tendermint-2846",
+      "connection_id": "cconnection-2358"
+    },
+    "channels": [
+      {
+        "chain_1": {
+          "channel_id": "channel-3",
+          "port_id": "transfer"
+        },
+        "chain_2": {
+          "channel_id": "channel-641",
+          "port_id": "channel-1279"
```

Comment:
this port_id should be 'transfer', right?

Reply:
ah that's right. updated thanks!